packages/kube-config/lib/index.js

```diff
@@ -152,11 +152,17 @@ function fromKubeconfig (input) {
 }
 
 function dumpKubeconfig ({ user, context = 'default', cluster = 'garden', namespace, token, server, caData }) {
+  const certificateAuthorityData = caData
+    ? {
+        'certificate-authority-data': caData
+      }
+    : {}
+
```

Comment:
```suggestion
  const cluster = {}
  if (caData) {
    cluster['certificate-authority-data'] = caData
  }
```

Reply:
And we should rename the function arguments to userName, contextName and clusterName to avoid missunderstandings.